legend-engine-xt-documentStore-pure/src/main/resources/core_documentstore/pureToDocumentStoreQuery/pureToDocumentStoreQuery.pure

```diff
@@ -1197,8 +1302,19 @@ function meta::external::store::document::functions::pureToDocumentStoreQuery::b
    if($uniqueName->length() > $maxSize, |$uniqueName->substring(0, $maxSize - 35) + '___' + $uniqueName->substring($maxSize - 35)->meta::pure::functions::hash::hash(meta::pure::functions::hash::HashType.MD5), |$uniqueName);
 }
 
-
-
+function meta::external::store::document::functions::pureToDocumentStoreQuery::addPlaceHoldersForLambdaOpenVariables(fn:FunctionDefinition<Any>[1], vars:Map<VariableExpression, ValueSpecification>[1], state:State[1]):State[1]
```

Comment:
this one to be removed as well

Reply:
can wait for next MR, as long as the build doesn't break